extensions/positron-python/README.md

```diff
@@ -47,11 +47,11 @@ pyright
 Install the test requirements that are used in CI:
 
 ```sh
-pip install -r ../build/pinned-test-requirements.txt
+pip install -r python_files/positron/pinned-test-requirements.txt
 ```
 
 Run Positron's unit tests with [pytest](https://docs.pytest.org/en/8.0.x/):
 
 ```sh
-pytest python_files/positron/
```

Comment:
Just curious, did pytest not work for you?

Reply:
It was trying to use a different python version than `which python` was giving me. Wasim suggested this change.